pkg/util/testing/wrappers.go

```diff
@@ -191,6 +192,20 @@ func (w *WorkloadWrapper) PriorityClass(priorityClassName string) *WorkloadWrapp
 	return w
 }
 
+func (w *WorkloadWrapper) RuntimeClass(name string) *WorkloadWrapper {
+	if len(w.Spec.PodSets) > 0 {
+		w.Spec.PodSets[0].Spec.RuntimeClassName = &name
+	}
+	return w
+}
+
+func (w *WorkloadWrapper) Overhead(resources corev1.ResourceList) *WorkloadWrapper {
```

Comment:
Remove this if it's not being used.

Reply:
It's actually uesd.

Reply:
I can't find any usage.

Reply:
Here's  where it locates at , but since I'll remove this test,  I'll remove this also. https://github.com/kubernetes-sigs/kueue/blob/342578788b7bb20941b80d6959e2acee993f1570/test/integration/controller/core/workload_controller_test.go#L249-L259